extension/app/ts/components/simulationExplaining/SimulationSummary.tsx

```diff
@@ -420,9 +421,9 @@ export function SimulationSummary(param: SimulationSummaryParams) {
 	const ownAddresses = Array.from(summary.entries()).filter( ([address, _balanceSummary]) =>
 		param.simulationAndVisualisationResults.addressMetadata.get(address)?.metadataSource === 'addressBook' || BigInt(address) === param.simulationAndVisualisationResults.activeAddress
 	)
-	const notOwnAddresses = Array.from(summary.entries()).filter( ([address, _balanceSummary]) =>
-		param.simulationAndVisualisationResults.addressMetadata.get(address)?.metadataSource !== 'addressBook' && BigInt(address) !== param.simulationAndVisualisationResults.activeAddress
-	)
+	const notOwnAddresses = Array.from(summary.entries())
+		.filter( ([address, _balanceSummary]) => param.simulationAndVisualisationResults.addressMetadata.get(address)?.metadataSource !== 'addressBook' && BigInt(address) !== param.simulationAndVisualisationResults.activeAddress)
+		.filter( ([address, _balanceSummary]) => CHAINS[param.simulationAndVisualisationResults.chain].eth_donator !== BigInt(address) ) // remove eth donator
```

Comment:
Will this cause any problems if someone wants to actually simulate as the donor address for some reason?

Reply:
no, it will show correctly if they are simulating AS the donor (as we are just removing them from `notOwnAddress`). However, if you send funds to the donor, or otherwise interact with them, we are not showing changes to donor address.

Reply:
Is it hard to fix that (so sending to donor works correctly)?  If not, maybe just an edge case we shouldn't worry about.

Reply:
would add quite a bit complexity. I guess you mean that we would show tokens and eth send's but just hide the "-200k eth"?

Reply:
Yeah, ideally we would *only* hide the make-me-rich transaction and nothing else.